### PR TITLE
Fix processing of command history new changes

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
See [tesseact_ros PR](https://github.com/ros-industrial-consortium/tesseract_ros/pull/34) and [tesseract_ros issue](https://github.com/ros-industrial-consortium/tesseract_ros/issues/12)

### CODE REVIEW

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp [hunk 1]
```diff
@@ -480,7 +480,7 @@ bool EnvironmentWidget::getEnvironmentChangesCallback(tesseract_msgs::GetEnviron
   res.id = tesseract_->getEnvironment()->getName();
   res.revision = tesseract_->getEnvironment()->getRevision();
   const tesseract_environment::Commands& commands = tesseract_->getEnvironment()->getCommandHistory();
-  for (int i = (req.revision - 1); i < commands.size(); ++i)
+  for (int i = (req.revision == 0 ? 0 : req.revision); i < commands.size(); ++i)
```
Comment (schornakj) [inline tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp:483]: Is it necessary to do this as a ternary? I think that doing `int i = req.revision` has the same effect.

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp [hunk 2]
```diff
@@ -525,7 +525,7 @@ void EnvironmentWidget::newTesseractStateCallback(const tesseract_msgs::Tesserac
   if (state_msg->revision > current_version)
   {
     std::vector<tesseract_msgs::EnvironmentCommand> commands;
-    for (int i = (current_version == 0 ? 0 : current_version - 1); i < state_msg->revision; ++i)
+    for (int i = (current_version == 0 ? 0 : current_version); i < state_msg->revision; ++i)
```
Comment (schornakj) [inline tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp:528]: Same comment as at 483 above.

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp (hunk 1 above)
Comment (gavanderhoorn) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp:483]: I believe that's also what @IKapitaniuk suggested in https://github.com/ros-industrial-consortium/tesseract_ros/issues/12#issuecomment-677722662.

Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp:483]: Nope.

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp (hunk 2 above)
Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp:528]: Nope.

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp (hunk 1 above)
Comment (gavanderhoorn) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp]: `req.revision` can be either `0` or `> 0`, no?

Isn't that what the ternary evaluates to?

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp (hunk 2 above)
Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp]: fixed.

File: tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp (hunk 1 above)
Comment (Levi-Armstrong) [reply tesseract_ros/tesseract_rviz/src/render_tools/environment_widget.cpp]: fixed.

**Pull request merged**